### DISCUSSION TITLE
videos/vue/vue-echarts-integration/

### DISCUSSION THREAD

**Discussion by giscus** [zxuqian/zxuqian.cn#64] [Announcements]
# videos/vue/vue-echarts-integration/

在上一家公司里，公司的产品用到了大量的图表进行数据可视化，当时公司决定使用的是 Echarts，它是一个与框架无关的 JS 图表库，所以需要自行封装到 react 或 vue 中。

https://zxuqian.cn/videos/vue/vue-echarts-integration/

**Comment by mixiaoluo88** [comment]
感谢博主，非常有用！
